### PR TITLE
Added 503 status code to allocate job chunk response

### PR DESCRIPTION
**Pull request by RachelTucker**

### CODE REVIEW

File: ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/HeadBucketResponse.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ private void setStatus(final int statusCode) {
             case 403: this.status = Status.NOTAUTHORIZED; break;
             case 404: this.status = Status.DOESNTEXIST; break;
             default: {
-                LOG.error("Unexpected status code: " + Integer.toString(statusCode));
```
Comment (DenverM80) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/HeadBucketResponse.java]: didn't we already do this?  I better go make sure that commit got merged to 3_2_autogen branch

**Comment by DenverM80** [comment]
Looks good.  Checking to make sure the logging changes got merged to 3_2_autogen.

File: ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/HeadBucketResponse.java (hunk 1 above)
Comment (rpmoore) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/HeadBucketResponse.java]: It's possible it wasn't added to the autogen until recently.

Comment (DenverM80) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/HeadBucketResponse.java]: I verified that the commit in question got applied to both branches.

**Pull request merged**